### DISCUSSION TITLE
I need some help connecting remotely

### DISCUSSION THREAD

**Discussion by David1946** [GerryDazoo/Slinger#500] [Q&A]
When I was last away from home in November last year I was able to connect remotely to my Slingbox at home using slingbox-server.py. Now when I try connecting I am getting an error message saying "error connecting to "DDNSaddress:5306". When I log on remotely to a computer at home and run slingbox_server.py it finds the slingbox I want on the local network reporting:-

 "Slingbox Found 10.0.1.121 5306 " labmFVP " FinderID "finder id""

Other apps that use the same DDNSaddress work perfectly.

Any suggestions as to what the problem might be would be much appreciated.

Many Thanks

**Comment by GerryDazoo** [comment]
5306 is the port number for the slingbox not the slingbox_server. Please check the [SERVER] port= in your config.ini and use that AND double check you've got the correct port forwarding rule in your router.

**Reply by David1946** [reply]
Many thanks Gerry. When I checked my port forwarding rules they needed to be updated the server now seems to be running and says:-
Host: 172.16.1.104:8080
Icy-MetaData: 1

01/18/2023, 22:33:44   Streaming connection from ('172.16.1.104', 51024)  
01/18/2023, 22:35:51   Streaming connection from ('172.16.1.104', 51025)
01/18/2023, 22:36:11   Streaming connection from ('172.16.1.104', 51027)

when I try using VLC or MPV using the URL http://172.16.1.104:8080/slingbox I get nothing.

Any ideas?

**Reply by David1946** [reply]
Please ignore that last message Gerry I've just got it working. Many Thanks
